### PR TITLE
ba-50 Create inventory item: Validate input

### PR DESCRIPTION
**Pull request by lyio**
Pre-select items in both dropdowns
Include auto-validate and required attributes

### CODE REVIEW

**Comment by lyio** [comment]
@flashback2k14

File: socobo-inventory-create.html [hunk 1]
```diff
@@ -46,13 +50,35 @@
...
+          class="input" 
+          id="countInput" 
+          value="{{item.count}}" 
+          label="Menge/Anzahl"
+          error-message="Zahl erforderlich"
+          maxlength="5"
+          minlength="1"
+          auto-validate
+          required></paper-input>
+          <paper-dropdown-menu label="Art" class="kind-dropdown" id="measurementDropdown">
```
Comment (flashback2k14) [inline socobo-inventory-create.html:78]: paper-input and paper-dropdown-menu not matching the input line

**Comment by lyio** [comment]
Hopefully fixed the alignment
alignment

**Comment by flashback2k14** [comment]
nice! 
i.O.

**Pull request merged**